package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "axios-timing-plugin",
+    "version": "1.0.0",
+    "description": "Axios plugin to measure the actual time it takes for a request to complete",
+    "main": "src/index.js",
+    "scripts": {
+        "test": "./node_modules/.bin/_mocha ./tests/test"
```

Comment:
`./node_modules/.bin/_mocha` should be `$(npm bin)/mocha`

Reply:
$(npm bin)/mocha